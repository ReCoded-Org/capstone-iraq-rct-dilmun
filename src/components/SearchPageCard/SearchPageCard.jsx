import React from 'react'

export default function SearchPageCard() {
    return (
        <div>
            hello
        </div>
    )
}
