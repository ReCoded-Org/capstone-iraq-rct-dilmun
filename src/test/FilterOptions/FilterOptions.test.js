import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import SearchPageCard from '../../components/SearchPageCard'

it('Test: Filter Option Passed', () => {
  const tree = renderer
    .create(
      <Router>
        {' '}
        <SearchPageCard />{' '}
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
