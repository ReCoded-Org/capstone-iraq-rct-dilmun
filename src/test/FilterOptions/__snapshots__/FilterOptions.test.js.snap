// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: Filter Option Passed 1`] = `
Array [
  " ",
  <div
    className="w-52 h-72 bg-white shadow-md mx-6 mb-10 relative rounded-md transform hover:scale-105 hover:shadow-xl transition duration-300 ease-in-out"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="link"
    tabIndex={0}
  >
    <img
      alt="car"
      className="bg-contain bg-center rounded-t-md"
      src="profile_bg.png"
    />
    <div
      className="absolute bg-orange top-0 left-0 px-2 rounded-tl-md"
    >
      
    </div>
    <div
      className="font-bold my-2 ml-1"
    >
      
    </div>
    <div
      className="flex justify-between items-center ml-1 space-x-2"
    >
      <p
        className="text-xs h-12 overflow-ellipsis overflow-hidden "
      >
        
      </p>
      <div
        className="bg-yellow text-xs font-black px-3 rounded-l py-1.5"
      >
        0
      </div>
    </div>
    <div
      className="text-xs flex justify-between px-2 pb-2 absolute inset-x-0 bottom-0"
    >
      <div
        className="flex space-x-0.5"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-eye fa-w-18 "
          data-icon="eye"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <p
          className=""
        >
          0
        </p>
      </div>
      <div
        className="flex space-x-0.5"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-clock fa-w-16 "
          data-icon="clock"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M256,8C119,8,8,119,8,256S119,504,256,504,504,393,504,256,393,8,256,8Zm92.49,313h0l-20,25a16,16,0,0,1-22.49,2.5h0l-67-49.72a40,40,0,0,1-15-31.23V112a16,16,0,0,1,16-16h32a16,16,0,0,1,16,16V256l58,42.5A16,16,0,0,1,348.49,321Z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <p>
          
        </p>
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-map-marker-alt fa-w-12 "
          data-icon="map-marker-alt"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 384 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M172.268 501.67C26.97 291.031 0 269.413 0 192 0 85.961 85.961 0 192 0s192 85.961 192 192c0 77.413-26.97 99.031-172.268 309.67-9.535 13.774-29.93 13.773-39.464 0zM192 272c44.183 0 80-35.817 80-80s-35.817-80-80-80-80 35.817-80 80 35.817 80 80 80z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <p>
          
        </p>
      </div>
    </div>
  </div>,
  " ",
]
`;
